plugins/nemo-safe-synthesizer/scripts/setup_model_filesets.py

```
@@ -119,15 +114,15 @@ def create_filesets(
                 purpose="generic",
                 storage=fileset_config["storage"],
             )
-            logger.info(f"Created fileset: {full_name}")
+            logger.info("Created fileset: %s", full_name)
             created.append(name)
         except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)